### ISSUE TITLE
[MHV MR / Accessibility] List of label/value pairs provided using headings

### ISSUE THREAD

**Issue by sarahhortonadhoc** [department-of-veterans-affairs/va.gov-team#123505]
### 📝 User story

As a screen reader user, I want correctly identified heading elements, so that I can use heading navigation and the heading list feature to understand and navigate information structures.

As a screen reader user, I want correctly identified list elements, so that I know how may items are in a list and can navigate the list.

### 📝 Problem statement

Label and value pairs (e.g., “Location: Dayton”) are marked up using heading and paragraph elements.

### 🌌 Supporting screenshots or videos

<img width="567" height="326" alt="Screenshot of a list of label/value pairs with an overlay showing the labels are marked up as h2 elements" src="https://github.com/user-attachments/assets/e0eac088-8967-4657-9615-f3f6c4340cfe" />

```
<h3…>Location</h3>…
<p…>DAYTON</p>…
<h3…>Written by</h3>…
<p …>MARUF AHMED</p>
```

### 👣 Reproduction steps

1. Navigate to the Medical Records > Care summaries and notes list page.
2. Navigate to a details page (e.g., “ADHC Consult Results”).
3. Scroll to the section that follows the Details heading.
4. Examine the rendered properties of the details.
5. Notice the use of `<h3>` and `<p>` elements.

### 🔗 Point of contact

@sarahhortonadhoc

### ⚠️ Severity level

3, Moderate. Should be fixed in 1-3 sprints post-launch.

#### Accessibility Standards
- [Guideline 1.3 Adaptable: Create content that can be presented in different ways (for example simpler layout) without losing information or structure.](https://www.w3.org/WAI/WCAG22/Understanding/adaptable)
  - [1.3.1 Info and Relationships (Level A)](https://www.w3.org/WAI/WCAG22/Understanding/info-and-relationships.html)

#### Experience Standards
- [Category Usability: User is able to easily know or learn how to use the interface to complete tasks.](https://design.va.gov/about/experience-standards/#category-usability)
  - [User can identify an element and its state](https://design.va.gov/about/experience-standards/#user-can-identify-an-element-and-its-state)

### 𝌣 Proposed solution or next steps

Use the Description List `<dl>` and associated elements, for example:

```
<dl>
  <dt>Location</dt>
  <dt>DAYTON</dd>
  <dt>Written by</dt>…
  <dt>MARUF AHMED</dt>
  …
</dl>
```

### ✅ Acceptance criteria

- [ ] The system displays the list of label/value pairs using description list markup.
- [ ] The screen reader user hears the list of label/values pairs announced as a list with [x] items.

### ✅ Possible tasks

- [ ] Review, refine, and acknowledge feedback.
- [ ] Update ticket information (title, assignee, labels, project(s), etc.)
- [ ] Fix and/or document decisions made.
- [ ] Product manager to move ticket in their respective product board to 'QA/A11y Review' after reviewing documented decisions / validating fix.
- [ ] Accessibility specialist will close the ticket after reviewing documented decisions / validating the fix.
